build.sbt

```diff
@@ -175,9 +205,13 @@ lazy val tests = crossProject(JVMPlatform, JSPlatform, NativePlatform)
   )
   .jvmConfigure(
     _.enablePlugins(GraalVMNativeImagePlugin)
-      .dependsOn(typesafe, sconfigJVM, docs)
```

Comment:
I don't think we can depend on docs anymore given it has no Scala 3 version.

I was getting lots of errors during update, so rage quit and moved the 1 test docs has into its own module

Reply:
Would sbt-projectmatrix solve this problem? If the Scala 2 test project could depend on the docs project then we could keep the docs test in the same `testsJVM` project 🤔 

It's totally fine to separate the tests like in this PR, I'm just curious.

Reply:
Wondered that myself, but I think this is fundamental.

Matrix for `docs` will contain `docs_2.13` and `docs_2.12`  (with different names)
Matrix for `testsJVM` will contain `testsJVM_2.13`, `testsJVM_2.12`, `tests_3`

When we say `testsJVM.dependsOn(docs)` projectmatrix will have to construct all the links statically

`testsJVM_2.12` -> `docs_2.12`
`testsJVM_2.13` -> `docs_2.13`
`testsJVM_3` -> ???

sbt-projectmatrix only aligns the right cells in matrics by platform and scala version, but if there's a hole, it will loudly complain.

We can do this by making a `docs_3` an empty project with `publish/skip := true` and only Scala 2 sources, but it might be overkill.

I'm looking into helping folks publish scalatags for Scala 3, so this should be but a temporary restriction